composer.json

```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "php": ">=5.6.0",
+        "php": "^8.0",
```

Comment:
```suggestion
        "php": "^7.4|^8.0",
```

Reply:
This will only apply if we keep the `mixed` types, and not embrace union types.